Assets/Vive-Teleporter/Scripts/TeleportVive.cs

```diff
@@ -78,8 +78,13 @@ void Start()
         if (GetChaperoneBounds(out p0, out p1, out p2, out p3))
         {
             BorderPointSet p = new BorderPointSet(new Vector3[] {
-                    p0, p1, p2, p3, p0
-                });
+                // Rotate to match camera rig rotation
+                Matrix4x4.TRS(Vector3.zero, OriginTransform.rotation, Vector3.one) * p0,
```

Comment:
Move `Matrix4x4.TRS(Vector3.zero, OriginTransform.rotation, Vector3.one)` to its own variable, so that it isn't calculated 4 times.